.github/workflows/ci.yaml

```diff
@@ -47,6 +52,8 @@ jobs:
           username: ${{ github.repository_owner }}
           password: ${{ secrets.GITHUB_TOKEN }}
           registry: ghcr.io
+      - name: Set up QEMU
```

Comment:
Just copy from OpenElis project , like they are using qemu before docker buildx 

Worry about why it is failing 🥲